crates/rpcsrv/Cargo.toml

```diff
@@ -27,3 +27,4 @@ tonic = { workspace = true }
 bytes = "1.4"
 futures = "0.3.29"
 dashmap = "5.5.0"
+arrow-flight = { version = "47.0.0", features = ["flight-sql-experimental"] }
```

Comment:
This must be locked to the same version of arrow that datafusion is using. Hence using `47.0.0` instead of the latest (`49.0.0`)